### DISCUSSION TITLE
Purple image only

### DISCUSSION THREAD

**Discussion by Marchal1977** [aaronwmorris/indi-allsky#1438] [General]
Hi,

Having a problem with my Raspberry HQ camera, whatever I change in the WebUI, the camera image gives a purple output only.

The camera works well by checking libcamera-vid -t 0.

Any ideas?

Regards M

**Comment by aaronwmorris** [comment]
While indi-allsky is running, can you monitor the log at `/var/www/indi-allsky/indi-allsky.log` and see if you catch any exceptions.  In that log, it should also include the command being used to capture the images.  That would be helpful.

Can you also post the the support info report here?

**Comment by Marchal1977** [comment]
Hi,

 

For some reason it showed a proper image, so far it looks good, I will monitor and report back when the problem re-occurs.

 

Thanks!

 

Van: Aaron W Morris ***@***.***> 
Verzonden: maandag 5 augustus 2024 22:14
Aan: aaronwmorris/indi-allsky ***@***.***>
CC: Marchal1977 ***@***.***>; Author ***@***.***>
Onderwerp: Re: [aaronwmorris/indi-allsky] Purple image only (Discussion #1438)

 

While indi-allsky is running, can you monitor the log at /var/www/indi-allsky/indi-allsky.log and see if you catch any exceptions. In that log, it should also include the command being used to capture the images. That would be helpful.

Can you also post the the support info report here?

**Reply by aaronwmorris** [reply]
Excellent!
